TACC/tup-ui
# fix(cms): tup-339 logos

## Overview

- Use white TACC logo.
- Remove TACC from branding.
- Add NSF to branding.

## Related

- [TUP-339](https://jira.tacc.utexas.edu/browse/TUP-339)

## Changes

- change branding and load new logo
- rename existing svg logo
- add white svg logo

## Testing

1. Open https://dev.tup.tacc.utexas.edu .
2. Verify [TUP-339](https://jira.tacc.utexas.edu/browse/TUP-339) expectations are met.

## UI

| Local | Remote |
| - | - |
| ![TUP-339 Local](https://user-images.githubusercontent.com/62723358/195724888-9f126444-919a-4b12-97b9-8b70d84549c6.png) | ![Remote TUP-339](https://user-images.githubusercontent.com/62723358/195950526-63e7125f-9073-483c-bde6-d1b34f35de57.png) |
